alicloud/resource_alicloud_security_group_test.go

```diff
@@ -223,3 +257,44 @@ resource "alicloud_vpc" "vpc" {
   cidr_block = "10.1.0.0/21"
 }
 `
+
+const testAccCheckSecurityGroupConfigTags = `
+variable "name" {
+  default = "tf-testAccCheckSecurityGroupConfigTags"
+}
+resource "alicloud_security_group" "foo" {
+  name = "${var.name}"
+  vpc_id = "${alicloud_vpc.vpc.id}"
+  tags {
+		foo = "bar"
+		bar = "foo"
+  }
+}
+
+resource "alicloud_vpc" "vpc" {
+  name = "${var.name}"
+  cidr_block = "10.1.0.0/21"
+}
+`
+const testAccCheckSecurityGroupConfigTagsUpdate  = `
+variable "name" {
+  default = "tf-testAccCheckSecurityGroupConfigTagsUpdate"
+}
+resource "alicloud_security_group" "foo" {
+  name = "${var.name}"
+  vpc_id = "${alicloud_vpc.vpc.id}"
+  tags {
+		bar1 = "zzz"
+		bar2 = "bar"
+		bar3 = "bar"
+		bar4 = "bar"
+		bar5 = "zzz"
+		bar6 = "bar"
+  }
+}
+
+resource "alicloud_vpc" "vpc" {
+  name = "${var.name}"
+  cidr_block = "10.1.0.0/21"
+}
+`
```

Comment:
Please add `tags` guide in the website/docs/r/security_group.html and website/docs/d/security_groups.html

Reply:
done